pkgs/servers/photoprism/generate-dependencies.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix
+
+node2nix \
+    --node-env ./node-env.nix \
+    --lock _package-lock.json \
```

Comment:
`_package-lock.json` was not included in this PR - not sure if that is intentional or not.

Reply:
Oops, my mistake